doc/SUPPORTED_PLATFORMS.rst

```diff
@@ -0,0 +1,61 @@
+.. _supported-platforms-inference:
+
+Supported platforms for inference
+=================================
+
+Here we maintain the list of supported platforms for running inference.
+
+Linux / AMD64 without GPU
+^^^^^^^^^^^^^^^^^^^^^^^^^
+* x86-64 CPU with AVX/FMA (one can rebuild without, but it might slow processing)
+* Ubuntu 14.04+ (glibc >= 2.19, libstdc++6 >= 4.8)
+* Full TensorFlow runtime
+* TensorFlow Lite runtime
+
+Linux / AMD64 with GPU
+^^^^^^^^^^^^^^^^^^^^^^
+* x86-64 CPU with AVX/FMA (one can rebuild without, but it might slow processing)
+* Ubuntu 14.04+ (glibc >= 2.19, libstdc++6 >= 4.8)
+* CUDA 10.0 (and capable GPU)
+* Full TensorFlow runtime
+* TensorFlow Lite runtime
+
+Linux / ARMv7
+^^^^^^^^^^^^^
+* Cortex-A53 compatible SoC ARMv7 + NEON
+* Raspbian Buster-compatible distribution
+* TensorFlow lite runtime
+
+Linux / Aarch64
+^^^^^^^^^^^^^^^
+* Cortex-A72 compatible SoC Aarch64
+* ARMbian Buster-compatible distribution
+* TensorFlow lite runtime
+
+Android / ARMv7
+^^^^^^^^^^^^^^^
+* SoC ARMv7 + NEON
+* Android 7.0-10.0
+* NDK API level >= 21
+* TensorFlow lite runtime
+
+Android / Aarch64
+^^^^^^^^^^^^^^^^^
+* SoC Aarch64
+* Android 7.0-10.0
+* NDK API level >= 21
+* TensorFlow lite runtime
+
+macOS / AMD64
+^^^^^^^^^^^^^
+* x86-64 CPU with AVX/FMA (one can rebuild without, but it might slow processing)
+* macOS >= 10.10
+* Full TensorFlow runtime
+* TensorFlow lite runtime
+
+Windows / AMD64
```

Comment:
On the Linux section you separated without GPU and with GPU. Why not here too?

Reply:
because I forgot we had GPU support on Windows :)